static/templates/draft.hbs

```diff
@@ -35,6 +35,10 @@
                         <div class="draft_controls">
                             <i class="fa fa-pencil fa-lg restore-draft tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="{{t 'Restore draft' }} (Enter)"></i>
                             <i class="fa fa-trash-o fa-lg delete-draft tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="{{t 'Delete draft' }} (Backspace)"></i>
+                            <label class="checkbox">
+                                <input type="checkbox" id="mark-draft" class="mark-draft" aria-hidden="true"/>
```

Comment:
`id` attribute here would be same for every draft which it shouldn't be ideally. I guess this is not required as `#mark-draft` could easily be replaced with its class selector `.mark-draft`.